.travis.yml

```diff
@@ -18,7 +18,10 @@ matrix:
   include:
     - php: 5.5
     - php: 7.0
-      env: 
+      env:
+        - SYMFONY__DATABASE__DRIVER=pdo_pgsql
+        - SYMFONY__DATABASE__USER=postgres
+        - SYMFONY__DATABASE__PASSWORD=postgres
```

Comment:
The changes in runtests were introduced, because its more convenient to run the tests on postgres. As far as I know, this config file only has an impact on travis and not on my local machine, or does it?